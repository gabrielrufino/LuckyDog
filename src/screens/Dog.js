import React from 'react';
import {Text, View} from 'react-native';

export default function Searcher() {
  return (
    <View>
      <Text>Dog</Text>
    </View>
  );
}
