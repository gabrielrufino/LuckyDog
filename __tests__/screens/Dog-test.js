import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import Dog from '../../src/screens/Dog';

describe('screens/Dog.js', () => {
  it('Should renders the dog screen correctly', () => {
    const route = {
      params: {
        image: '',
      },
    };

    const tree = renderer.create(<Dog route={route} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
