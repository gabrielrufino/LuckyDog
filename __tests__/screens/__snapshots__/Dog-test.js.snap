// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`screens/Dog.js Should renders the dog screen correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "padding": 20,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#CACAAA",
        "borderColor": "#C8AB83",
        "borderRadius": 4,
        "borderWidth": 4,
        "flex": 1,
      }
    }
  >
    <Image
      resizeMode="cover"
      source={
        Object {
          "uri": "",
        }
      }
      style={
        Object {
          "borderRadius": 4,
          "height": "100%",
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "marginTop": 10,
      }
    }
  >
    <View
      style={
        Object {
          "marginRight": 10,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "color": "#EEC584",
              "fontSize": 30,
            },
            undefined,
            Object {
              "fontFamily": "FontAwesome5Free-Regular",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {
              "fontWeight": "400",
            },
          ]
        }
      >
        
      </Text>
    </View>
    <View
      style={
        Object {
          "marginRight": 10,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "#CACAAA",
                "fontSize": 30,
              },
              undefined,
              Object {
                "fontFamily": "FontAwesome5Free-Solid",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "fontWeight": "900",
              },
            ]
          }
        >
          
        </Text>
      </View>
    </View>
  </View>
</View>
`;
